src/app/screens/History/index.js

```diff
@@ -1,18 +1,37 @@
-import React from 'react';
-import { connect, useSelector } from 'react-redux';
+import React, { useEffect } from 'react';
+import { connect, useDispatch } from 'react-redux';
 import { UTLabel } from '@widergy/energy-ui';
 import i18 from 'i18next';
+import { any, objectOf } from 'prop-types';
+
+import NotesActions from 'redux/notes/actions';
 
 import NotesContainer from './components/NotesContainer';
 import styles from './styles.module.scss';
 
-const History = () => (
-  <div className={styles.container}>
-    <UTLabel bold large black>
-      {i18.t('History:title')}
-    </UTLabel>
-    <NotesContainer notes={useSelector(state => state.notes)} />
-  </div>
-);
+const History = ({ notes }) => {
+  const dispatch = useDispatch();
```

Comment:
About my comment above 

```suggestion
const History = ({ notes, dispatch }) => {
```